tests/client-dbus/tests/udev/test_start.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Test starting additional stratisd processes.
+"""
+
+# isort: STDLIB
+import os
+
+# isort: LOCAL
+from stratisd_client_dbus import Manager, Pool, get_object
+from stratisd_client_dbus._constants import TOP_OBJECT
+
+from ._utils import OptionalKeyServiceContextManager, UdevTest, create_pool
+
+
+class TestFailedStart(UdevTest):
+    """
+    Test creating an encrypted pool and stopping it, removing the key from the keyring,
+    and then start it.
+
+    This is a regression test for a bug.
+    """
+
+    def failed_start_regression_test(self):
```

Comment:
Change the name from ```failed_start_regression_test``` to ```test_failed_start...```. unittest's standard naming convention expects every actual test to have a ```test_``` prefix.